Changelog.md

```diff
@@ -2,40 +2,38 @@ Changelog AFWall+
 ==================
 
 Version 2.5.2
-* Logs should load much faster
-* Auto trim log database
-* Switch log viw bug
-* Crash on export and import
-* Re-moved Lock screen notification xposed,since it was draining battery
-* Updates languages
+* New Feature: Auto-trim log database
+* Enhancement: Logs should load much faster
+* Change: Move lock screen notification back to Xposed, since it was draining battery
```

Comment:
LOL... the dash in "re-moved" had me thinking you moved it again! :-)

I'll correct it promptly.